lib/Dialect/CKKS/IR/CKKSAttributes.td

```diff
@@ -0,0 +1,40 @@
+#ifndef LIB_DIALECT_CKKS_IR_CKKSATTRIBUTES_TD_
+#define LIB_DIALECT_CKKS_IR_CKKSATTRIBUTES_TD_
+
+include "lib/Dialect/CKKS/IR/CKKSDialect.td"
+
+include "mlir/IR/AttrTypeBase.td"
+include "mlir/IR/DialectBase.td"
+include "mlir/IR/OpBase.td"
+
+class CKKS_Attribute<string attrName, string attrMnemonic>
+    : AttrDef<CKKS_Dialect, attrName> {
+    let summary = "Attribute for CKKS";
+    let description = [{
+      This attribute represents the values for CKKS.
+    }];
```

Comment:
```suggestion
```

Since it's a base class, the def should always override these fields. They can be removed